js/scorm/wrapper.js

```diff
@@ -100,6 +106,11 @@ define (function(require) {
 	ScormWrapper.prototype.initialize = function() {
 		this.lmsConnected = this.scorm.init();
 
+		// the pipwerks code will set the course to 'incomplete' automatically, but it doesn't commit that change
+		if(this.scorm.handleCompletionStatus && this.commitOnStatusChange && this.isFirstSession()) {
+			this.commit();
+		}
+
```

Comment:
this if() statement can now be removed as the updated pipwerks code does now call commit when the setting the course to incomplete